### DISCUSSION TITLE
Thảo luận

### DISCUSSION THREAD

**Discussion by ducialtit1** [ducialtit1/OOP---group-20---N03#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ducialtit1** [comment]
hello

**Comment by ducialtit1** [comment]
@Yvyvibtr

**Comment by ducialtit1** [comment]
làm thế nào giờ

**Comment by Yvyvibtr** [comment]
câu 2 chắc phải tách ra

**Comment by Yvyvibtr** [comment]
lm ý 2 trc đi

**Comment by Yvyvibtr** [comment]
ý 1 từ âm vô cùng đến dương vô cùng thì chay sao =))
